crm/types.go

```diff
@@ -58,4 +82,130 @@ type (
 		Color     string `json:"color"`
 		ID        int    `json:"id"`
 	}
+
+	// CRM EntityResponse, returns when entity object was created successfully
+	EntityResponse struct {
+		Id               int           `json:"id"`
+		CreatedAt        int           `json:"created_at"`
+		UpdatedAt        int           `json:"updated_at"`
+		Title            string        `json:"title"`
+		Owner            int           `json:"owner"`
+		CreatedBy        int           `json:"created_by"`
+		ChangedBy        string        `json:"changed_by"`
+		Converted        bool          `json:"converted"`
+		FirstName        string        `json:"first_name"`
+		LastName         string        `json:"last_name"`
+		Patronymic       string        `json:"patronymic"`
+		Companies        []Company     `json:"companies"`
+		Position         string        `json:"position"`
+		UtmSource        string        `json:"utm_source"`
+		UtmMedium        string        `json:"utm_medium"`
+		UtmCampaign      string        `json:"utm_campaign"`
+		UtmContent       string        `json:"utm_content"`
+		UtmTerm          string        `json:"utm_term"`
+		Messengers       string        `json:"messengers"`
+		Phone            string        `json:"phone"`
+		Email            string        `json:"email"`
+		Comments         string        `json:"comments"`
+		Source           []Source      `json:"source"`
+		ContactLabel     []Label       `json:"contact_label"`
+		Ownercopy        string        `json:"ownercopy"`
+		LastNamecopy     string        `json:"last_namecopy"`
```

Comment:
what are these `copy` fields?

Reply:
it's fields that returns when entity was created